src/app/user.ts

```diff
@@ -0,0 +1,6 @@
+import {oauth} from '../store'
+
+export function isUserConnected(userID: string): boolean {
```

Comment:
Add isSysAdmin() function.  Will be used for subscriptions, configuration options. 
 
* run slash command to save client host, secret